release-notes/release-notes57.md

```diff
@@ -37,7 +37,9 @@ The following new features were added for the {{site.productname}} 5.7 release.
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
-### Enhancement name
+### Table widths are now retained when copying and pasting tables
+
+Tables copied to the clipboard when using the `table` plugin will now retain their width and the width of each column when pasted.
```

Comment:
This doesn't quite sound right to me. The table width won't be retained, instead it'll be adjusted to match the size of the copied columns. So maybe something like this instead?

```suggestion
Tables copied to the clipboard, when using the `table` plugin, will now correctly retain the width of each column and adjust the table width to match the column widths when pasted.
```

Reply:
This was a difficult one. I chose the wording based on the assumption that the subset of the rows/columns that you've copied are (from an ultra generalised perspective) a new table when you copy them.

I agree it needs work but I don't think this works either, particularly: "adjust the table width to match the column widths when pasted." because technically the table width is the sum of the column widths.

Needs more work, for sure, I'm keen for @tylerkelly13's advice.

Reply:
```suggestion
Tables copied to the clipboard when using the `table` plugin will now correctly retain the width of each column. When pasted into another table, its width will be adjusted to account for the column widths.
```
or something? not great either but I thought that splitting the copying part and pasting part into two sentences might make things a little clearer?

Reply:
That's not quite correct either, as the table width is adjusted on copy.

Reply:
```suggestion
Tables copied to the clipboard when using the `table` plugin will now correctly retain their width and the widths of each of the copied columns.
```

Reply:
That has the same issue that I mentioned in the original comment, as it sounds like the table will remain the same size.

Reply:
```suggestion
Tables copied to the clipboard when using the `table` plugin will now correctly retain the width of each column. Additionally, the table width will be adjusted to account for the copied column widths.
```
Maybe?

Reply:
I'm happy with that ^